frontend/README.md

```diff
@@ -18,15 +18,16 @@ Write a frontend application in TypeScript that Fetches and displays
    2. Can sort asc/desc by `Investor Annual Interest Due`
    3. Can reset sorting
    4. We expect filters and sorting to be compatible with one another.
-5. Having a BoE (Bank of England) rate input field that should be applied on top of the investment account rate. Default BoE rate should be 2.25%
+4. Having a BoE (Bank of England) rate input field that should be applied on top of the investment account rate. Default BoE rate should be 2.25%
+5. The layout should be in a table, as per the below example:
 
 
-|                                  | `${investmentAccount} (${annualRate}%)`   | ... | Investor Total [^1] | Investor Annual Interest Due [^2] |
-|----------------------------------|-------------------------------------------|-----|---------------------|-----------------------------------|
-| `${investor.name}`               | `${holding.balance}`                      |     | £                   | £                                 |
-| ...                              |                                           |     |                     |                                   |
-| Account Total [^3]               | £                                         |     | £                   | £                                 |
-| Account Annual Interest Due [^4] | £                                         |     | £                   | -                                 |
+|                                  | `${investmentAccount} (${annualRate}%)`   | ... (remaining accounts) | Investor Total [^1] | Investor Annual Interest Due [^2] |
+|----------------------------------|-------------------------------------------|--------------------------|---------------------|-----------------------------------|
```

Comment:
Looks good. I think we need to provide a visual though, which I can make. Perhaps a ticket needs to be put on the board for this?

Reply:
Ticket on the board 👍